index.js

```diff
@@ -2,34 +2,34 @@
 const express = require('express');
 const bodyParser = require('body-parser');
 const app = express();
-const port = 3000;
+const port = 3001;
 const fs= require('fs');
 const data = require('./todo.json');
-const { v4: uuidv4 } = require('uuid');
-
 
 app.use(bodyParser.json())
 
 
 
-app.post('/todos', (req, res) => {
-
-    const { title,description,completed } = req.body
+app.get('/todos/:completed', (req, res) => {
+    const statusFilter = req.params.completed; 
+    const new_data = [];
 
-    const titles = data.map(d => d.title);
+    data.forEach(item => {
```

Comment:
instead of these lines you should write all in filter function as we discussed during the lesson